planet4

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I did the change :+1:  But aren't `go`, `stop` and `wait` also shell scripts?

I also moved it under a folder. Since this is a script that is not executed directly from the Makefile, to distinguish it from those that are.

Reply:
These scripts are probably the ONLY example of shell scripts in the planet4- namespace which are not `.sh` !